views/forms/TextInput/index.jsx

```diff
@@ -23,6 +23,11 @@ const TextInput = (props) => {
       size={props.size}
       disabled={props.disabled}
       {...field}
+      onChange={(e) => {
+        if (props.onChange) props.onChange(e.target.value)
```

Comment:
IMO we should call this with the event and let the handling function pull out what it needs.

```suggestion
        if (props.onChange) props.onChange(e)
```

Reply:
[216b66e](https://github.com/AMP-SCZ/dpdash/pull/560/commits/216b66e967a28fae64661657855e58bd0984fef3)